packages/protocol/src/protocol.yml

```diff
@@ -970,6 +970,22 @@ Browser:
 
     close:
 
+ConsoleMessage:
+  type: mixin
+  properties:
+    page: Page?
```

Comment:
I don't like that we made `page` optional for context console. Let's move it out of the mixin and make unconditional?